.github/workflows/build.yml

```diff
@@ -90,7 +90,6 @@ jobs:
         - /var/run/sas:/var/run/sas
     env:
       GCLIENT_EXTRA_ARGS: '--custom-var=checkout_mac=True --custom-var=host_os=mac'
-      PATCH_UP_APP_CREDS: ${{ secrets.PATCH_UP_APP_CREDS }}
```

Comment:
Don't we want this on both Mac and Linux, since we have separate cache keys for Mac and Linux?

Reply:
@VerteDinde iirc we initially decided to choose one or the other since patch changes _should_ cause the same index mismatch failure regardless